scene/main/window.cpp

```diff
@@ -893,12 +893,13 @@ void Window::_window_input(const Ref<InputEvent> &p_ev) {
 	}
 
 	if (exclusive_child != nullptr) {
+		/*
```

Comment:
What was the reasoning for this removal?

Reply:
I see no reason for it, and makes the built-in mode unusable

Reply:
Then why this code is commented out not removed? Such code is usually never used again and just litters the codebase.

Reply:
no, this will need more through inspection, I have an idea why it is there but it is causing problems, it needs to be replaced by something else, but prefer to do after this is merged.

Reply:
@reduz are you talking about #40227?